app/models/company_value.rb

```diff
@@ -1,5 +1,6 @@
 class CompanyValue < ApplicationRecord
   validates :title, presence: true, uniqueness: true
+  has_many :kudos, class_name: 'Kudo', dependent: :destroy
```

Comment:
You do not have to specify the class name, in such cases - RoR can guess it by itself :)
```suggestion
  has_many :kudos, dependent: :destroy
```